src/teleop_twist_joy.cpp

```diff
@@ -80,7 +80,7 @@ TeleopTwistJoy::TeleopTwistJoy(rclcpp::node::Node::SharedPtr & node)
   rmw_qos_profile_t custom_qos_profile;
   custom_qos_profile.history = RMW_QOS_POLICY_HISTORY_KEEP_LAST;
   custom_qos_profile.depth = 50;
-  custom_qos_profile.reliability = RMW_QOS_POLICY_RELIABILITY_RELIABLE;
+  custom_qos_profile.reliability = RMW_QOS_POLICY_RELIABILITY_BEST_EFFORT;
```

Comment:
So, I'm finding here that if I use the default sensor profile, the node fails to work about 25% of the times I fire it up.  I haven't done any debugging yet to figure out why that is, but that is much less reliable than what I have been using (BEST_EFFORT with a queue of 50).